lib/rocket_chat/messages/user.rb

```diff
@@ -38,6 +38,22 @@ def create(username, email, name, password, options = {})
       end
 
       #
+      # users.createToken REST API
+      # @param [String] user_id Rocket.Chat user id
+      # @param [String] username Username
+      # @return [Boolean]
+      # @raise [HTTPError, StatusError]
+      #
+      def create_token(user_id: nil, username: nil)
+        response = session.request_json(
+          '/api/v1/users.createToken',
+          method: :post,
+          body: user_params(user_id, username),
+        )
+        RocketChat::Token.new response['data']
```

Comment:
Shouldn't this just be `RocketChat::Token.new response` ? 

https://rocket.chat/docs/developer-guides/rest-api/users/createtoken